packages/cookie/src/index.ts

```diff
@@ -1 +1,6 @@
-export { Cookie, type CookieOptions } from './lib/cookie.ts'
+export {
+  Cookie,
+  type CookieOptions,
+  type CookieParseOptions,
+  type CookieSerializeOptions,
```

Comment:
`CookieParseOptions`/`CookieSerializeOptions` are re-exported out from here so the `@remix-run/session` package can get them without having a dep on `cookie-es`